client/lib/utils.js

```diff
@@ -22,6 +22,21 @@ Utils = {
     return string.charAt(0).toUpperCase() + string.slice(1);
   },
 
+  isMobile() {
```

Comment:
Where this function is actually used?

Anyway, even if we need to rely on the user-agent somewhere (which is usually a bad idea) we should use a [issmobile](https://atmospherejs.com/mquandalle/ismobile) package instead of using a custom (and in this case, a bit simple) regular expression to test it.